### PR TITLE
feat(Elasticsearch Node): Add bulk operations for Elasticsearch

### PR DESCRIPTION
**Pull request by pemontto**
## Summary
Add support for Elasticsearch [bulk operations](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html) to:
- Create (actually upsert)
- Update
- Delete

It defaults to 50 items per batch, but should be easy enough to make configurable. Depending on data size it could be reasonable to do batches of 100, 200, 500.

On 247 items the node time went from over a **minute (75s)** to **2s**.

## Old
image

## With _bulk
image

## Errors
It should handle item indices and error correctly
image

## Review / Merge checklist

- [X] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

**Review by Joffcom — APPROVED** [review]
This is great, Thanks.

**Comment by janober** [comment]
Got released with [`n8n@1.54.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.54.0)

File: packages/nodes-base/nodes/Elastic/Elasticsearch/GenericFunctions.ts [hunk 1]
```diff
@@ -20,13 +62,13 @@ export async function elasticsearchApiRequest(
...
-	const options: IRequestOptions = {
+	const options: IHttpRequestOptions = {
 		method,
 		body,
 		qs,
-		uri: `${baseUrl}${endpoint}`,
+		url: `${baseUrl}${endpoint}`,
 		json: true,
-		rejectUnauthorized: !ignoreSSLIssues,
+		skipSslCertificateValidation: ignoreSSLIssues,
```
Comment (netroy) [inline packages/nodes-base/nodes/Elastic/Elasticsearch/GenericFunctions.ts:71]: why was this line changed?
